src/views/User/User.tsx

```diff
@@ -956,6 +968,11 @@ export class User extends React.PureComponent<UserProperties, any> {
 
                     <h2>{_("Activity")}</h2>
                     <Card className="activity-card">
+                        {(!user.supporter && !user.is_moderator && !user.is_superUser) && <h4>{_("Vacation Accrued: (Non-Supporter)")}</h4>}
+                        {(user.supporter || user.is_moderator || user.is_superUser) && <h4>{_("Vacation Accrued: (Supporter)")}</h4>}
+                        {(user.on_vacation) && <div >{_("User On Vacation")}</div>}
+                        {(!user.on_vacation && (user.supporter || user.is_moderator || user.is_superUser)) && <div >{this.state.vacation_accrued_text} {_(" out of 60 Days")}</div>}
+                        {(!user.on_vacation && (!user.supporter && !user.is_moderator && !user.is_superUser)) && <div >{this.state.vacation_accrued_text} {_(" out of 30 Days")}</div>}
```

Comment:
@GreenAsJade suggested it would look less cluttered without displaying the rate of increase. So if you agree, merge this commit otherwise merge the previous commit to keep that info displayed.

Reply:
Cool, yeah agreed